chingu-voyages/v50-tier2-team-15
# Add Initial Project Setup and CI Workflows

This pull request includes the initial setup for our Vite-based React project, along with continuous integration workflows for building, testing, and linting the application. The CI workflows are set up to run on both the main and dev branches.

How to test:

Review the .github/workflows/ directory to see the workflow files.
Ensure that the workflows align with our project requirements and suggest any improvements.

![image](https://github.com/user-attachments/assets/b1f34ef8-74e0-4293-916b-b9479866343c)
